src/webui/webapplication.h

```diff
@@ -43,7 +43,7 @@
 #include "base/utils/net.h"
 #include "base/utils/version.h"
 
-constexpr Utils::Version<int, 3, 2> API_VERSION {2, 4, 1};
+constexpr Utils::Version<int, 3, 2> API_VERSION {2, 5, 0};
```

Comment:
Maybe we should preserve API version from previous release in comment near it? It will allow version management easier, IMO.

Reply:
It's easy enough to just use git for checking what the previous version was:
`git log -L 46,46:src/webui/webapplication.h`. Or, if the line number changes in the meantime: `git log -p -S "API_VERSION" -- src/webui/webapplication.h` or even `git log -p -S "API_VERSION" -- src/webui` (you get the idea). For those using IDEs, it's even less of a problem, since a lot of them allow showing the git history inline/side-by-side.
Introducing a comment will just introduce another place that has to changed every time.

Reply:
>It's easy enough to just use git for checking what the previous version was:

I don't propose to store previous API version. I propose to store API version from previous qBittorrent release.

Reply:
@glassez 
I see. There is this: `git log -L 46,46:src/webui/webapplication.h $(git rev-parse release-4.2.1)..HEAD --` but it only works correctly if you are checked out in the branch that both ends of the commit revision belong to. So for example executing the above on `master` rather than `v4_2_x` would not yield the "intuitive" result, nor would using `$(git rev-parse release-4.1.9)..HEAD --` on `4_2_x` or master. I'm sure there is way, but I am not a super expert in git.

I don't mind the decision either way.

Reply:
I suppose we can do it later as not to block this PR.